network/src/components/AppWrapper.js

```diff
@@ -735,6 +735,16 @@ export const AppWrapper = ({children}) => {
                   }}> 
                 Add Organization
               </button>
+              <button class='undo' onClick={() => {
+                  resetCanvasContextMenu();   // Reset the canvas context menu
```

Comment:
This is fine implementation for now since we don't have a way to track actions yet, but in the long run this function call will need to be changed as it only makes the contextMenu hidden if active.